src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -276,76 +307,152 @@ directly to CMake; for example::
 
 
 CMake Configuration Options
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------------------
+
+The tables in this section summarize the main build system options in Axom.
 
-Here are the key build system options in Axom:
+Axom components, tests, examples, etc.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 +------------------------------+---------+----------------------------------------+
 | OPTION                       | Default | Description                            |
 +==============================+=========+========================================+
-| AXOM_DEBUG_DEFINE            | DEFAULT | Controls whether the `AXOM_DEBUG`      |
-|                              |         | compiler define is enabled             |
-|                              |         |                                        |
-|                              |         | By DEFAULT, it is enabled for          |
-|                              |         | `Debug` and `RelWithDebInfo` configs   |
-|                              |         | but this can be overridden by setting  |
-|                              |         | `AXOM_DEBUG_DEFINE` to `ON` or `OFF`   |
-+------------------------------+---------+----------------------------------------+
 | AXOM_ENABLE_ALL_COMPONENTS   | ON      | Enable all components by default       |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_<FOO>            | ON      | Enables the axom component named 'foo' |
+| AXOM_ENABLE_<FOO>            | ON      | Enable the axom component named 'Foo'  |
 |                              |         |                                        |
 |                              |         | (e.g. AXOM_ENABLE_SIDRE)               |
-|                              |         | for the sidre component                |
+|                              |         | for the Sidre component                |
```

Comment:
How does `AXOM_ENABLE_ALL_COMPONENTS ON` interact with (say) `AXOM_ENABLE_SIDRE OFF`?  Does the more specific option control, or is it "last one wins?"

Reply:
It looks to me that that case will enable all components except Sidre. @white238 is that correct?

Reply:
That is what I would intend, and I think that's what we would get.  So could we say at this line in the table,

```suggestion
|                              |         | for the Sidre component.                |
|                              |         | Overrides AXOM_ENABLE_ALL_COMPONENTS.  |
```

Reply:
I added that to the table and a more explicit explanation in the text below.